ktor-server/ktor-server-plugins/ktor-server-partial-content/jvmAndNix/src/io/ktor/server/plugins/partialcontent/PartialContentUtils.kt

```diff
@@ -118,7 +118,7 @@ internal suspend fun BodyTransformedHook.Context.processMultiRange(
 ) {
     val boundary = "ktor-boundary-" + hex(Random.nextBytes(16))
 
-    call.attributes.put(SuppressionAttribute, true) // multirange with compression is not supported yet
+    call.suppressCompression() // multirange with compression is not supported yet
```

Comment:
Could you log an issue and link it in the comment? It would be also great to link this line in the issue